ui/receive_page.go

```diff
@@ -0,0 +1,594 @@
+package ui
+
+import (
+	"image"
+	"image/color"
+	"time"
+
+	"gioui.org/gesture"
+	"gioui.org/io/pointer"
+	"gioui.org/layout"
+	"gioui.org/op"
+	"gioui.org/op/paint"
+	"gioui.org/text"
+	"gioui.org/unit"
+	"gioui.org/widget"
+
+	"github.com/decred/dcrd/dcrutil"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/values"
+	"github.com/skip2/go-qrcode"
+	"golang.org/x/exp/shiny/materialdesign/icons"
+)
+
+const PageReceive = "Receive"
+
+type walletAccount struct {
+	selectAccount []*gesture.Click
+}
+
+type walletAccountWidget struct {
+	title                    decredmaterial.Label
+	walletAccountModal       *decredmaterial.Modal
+	walletsList              layout.List
```

Comment:
try to avoid using the type as part of the name of a variable or field. In this case, wallets is just fine.